.rubocop.yml

```diff
@@ -29,6 +29,9 @@ Style/AccessorGrouping:
     - lib/rubocop/formatter/base_formatter.rb
     - lib/rubocop/cop/offense.rb
 
+Style/ClassMethodsDefinitions:
```

Comment:
I'd still make this a pending cop and enable it in RuboCop's codebase using `self.method` as the style.